client/wc-api/wp-data-store/index.js

```diff
@@ -0,0 +1,45 @@
+/** @format */
+/**
+ * External dependencies
+ */
+import { registerGenericStore } from '@wordpress/data';
+
+/**
+ * Internal dependencies
+ */
+import createApiClient from './create-api-client';
+import wcApiSpec from '../wc-api-spec';
+
+function createWcApiStore() {
```

Comment:
This uses `apiFetch` which is multi-site capable inasmuch that it will work on whichever sub-site you're on at the time. For cross-site operations, it would require more work, but we don't have any existing UI for something like that anyway.